### DISCUSSION TITLE
Waterfall inline with RTA

### DISCUSSION THREAD

**Discussion by povnz** [g4ixt/QtTinySA#60] [Ideas]
Suggesting a feature that mimics the onscreen waterfall of the tinySA, so we have spectral history as well as the Realtime, the current implementation of the waterfull is nice, but a flat view inline with the exsiting freq scale is so much faster for diagnostics

**Comment by g4ixt** [comment]
I don't know how to create a waterfall plot, although there are some examples of [code in GitHub](https://github.com/flothesof/pyqtgraph-spectrographer?tab=readme-ov-file) that I might be able to clone.  I may have a look at this eventually.  The existing 'time spectrum' (3D) is poor and I would like to improve that some time if I can, but it's hard to understand surface plots and there are no nice features like axis scales for them in pyqtgraph.

**Comment by g4ixt** [comment]
Now in release v1.0.2
